src/sampleManifest.tsx

```diff
@@ -115,6 +116,7 @@ export const sampleManifest: SampleSpecGroup[] = [{
     getViewClipSpec(),
     getVolumeQuerySpec(),
     getZoomToElementsSpec(),
+    getScientificVisualizationSpec(),
```

Comment:
This list is supposed to be alphabetical based on SampleSpec.label.

Reply:
Updated